app/main/controllers/sources.py

```diff
@@ -1,8 +1,9 @@
 import json
 
 from flask import request, abort, jsonify
+from sqlalchemy import String, BOOLEAN
```

Comment:
Fix the BOOLEAN issue
```suggestion
from sqlalchemy import String, Boolean
```